test/fixtures_message_sender.js

```diff
@@ -1017,32 +1085,82 @@ module.exports = function() {
+                    "to_identity": "cb245673-aa41-4302-ac47-00000001002",
+                    "to_addr": "+27820001002",
+                    "content": "Please reply to this message with your compliment. If it relates to the service at the clinic, include the clinic or clinic worker name. Standard rates apply.",
+                    "metadata":{
+                      "template":{
+                        "name":"important_info",
+                        "variables":"Please reply to this message with your compliment. If it relates to the service at the clinic, include the clinic or clinic worker name. Standard rates apply."
+                      }
+                    },
+                    "channel":"pilot-channel"
+                  }
             },
-            'response': {}
+            "response": {
+                "code": 201,
+                "data": {}
+            }
         },
 
-        // 146:
+        // 146: create outbound message for cb245673-aa41-4302-ac47-00000001002
         {
-            'request': {
-                'method': 'GET',
-                'url': 'http://',
+            "key": "post.ms.outbound.27820001002",
+            "request": {
+                "url": 'http://ms/api/v1/outbound/',
+                "method": 'POST',
+                "data": {
+                    "to_identity": "cb245673-aa41-4302-ac47-00000001002",
+                    "to_addr": "+27820001002",
+                    "content": "Please reply to this message with your complaint. If it relates to the service at the clinic, include the clinic or clinic worker name. Standard rates apply.",
+                    "metadata":{
+                      "template":{
+                        "name":"important_info",
+                        "variables":"Please reply to this message with your compliment. If it relates to the service at the clinic, include the clinic or clinic worker name. Standard rates apply."
+                      }
+                    },
+                    "channel":"pilot-channel"
+                }
             },
-            'response': {}
+            "response": {
+                "code": 201,
+                "data": {}
+            }
         },
 
-        // 147:
-        {
-            'request': {
-                'method': 'GET',
-                'url': 'http://',
-            },
-            'response': {}
-        },
+
+        // 147: post outbound message for cb245673-aa41-4302-ac47-00000000001
+          {
+              "key": "post.ms.outbound.27820001002",
+              "request": {
+                  "url": 'http://ms/api/v1/outbound/',
+                  "method": 'POST',
+                  "data": {
+                      "to_identity": "cb245673-aa41-4302-ac47-00000000002",
+                      "to_addr": "+27820000111",
+                      "content": "HIV positive moms can have an HIV negative baby! You can get free medicine at the clinic to protect your baby and improve your health",
```

Comment:
Since we're using the `api.http.fixtures.add` to add the fixture in the test, we don't need this fixture here (unless there's another test that's not using the dynamic fixtures)